content/lessons/chapter-1/intro-1.tsx

```diff
@@ -7,6 +7,7 @@ export const metadata = {
   title: 'chapter_one.intro_one.title',
   image: '/assets/images/chapter-1-intro-1.jpg',
   theme: 'bg-[#2C3A3F]',
+  solidTheme: 'bg-[#2C3A3F]',
```

Comment:
Not sure I understand why we need a new `solidTheme`, could you elaborate a little @benalleng? What does it accomplish?

Reply:
The purpose of the solidTheme is mainly to solve a problem that arises with the use of tooltips
Our chapter themes are all gradients at the moment and future chapters also seems to be planned with gradient backgrounds, while this looks good with a gradual gradient across the entire background, for smaller elements like tooltipsI find the gradient unsightly

In our last call @GBKS discussed trying to remove any popups for transparent backgrounds but I am also considering trying to move towards image backgrounds instead of next/Image + transparent bg

perhaps we should use "primaryTheme" "secondaryTheme" monikers